netdata-installer.sh

```diff
@@ -394,6 +388,57 @@ fi
 # replace multiple spaces with a single space
 NETDATA_CONFIGURE_OPTIONS="${NETDATA_CONFIGURE_OPTIONS//  / }"
 
+NEED_PROTOBUF=true
+
+if [ -n "${USE_SYSTEM_PROTOBUF}" ]; then
+  NEED_PROTOBUF=false
+fi
+
+if [ -n "${NETDATA_DISABLE_CLOUD}" ] && [ -n "${NETDATA_DISABLE_PROMETHEUS}" ]; then
+  NEED_PROTOBUF=false
+fi
+
+if ${NEED_PROTOBUF} && [ "$(uname -s)" = "Linux" ] && [ -f /proc/meminfo ]; then
+  mega="$((1000 * 1000))"
+
+  if [ -n "${MAKEOPTS}" ]; then
+    proc_count="$(echo ${MAKEOPTS} | grep -oE '-j *[[:digit:]]+' | tr -d '-j ')"
```

Comment:
```suggestion
    proc_count="$(echo ${MAKEOPTS} | grep -oE '\-j *[[:digit:]]+' | tr -d '\-j ')"
```

Not sure why we need it, but the current code fails, see

- before

```cmd
[pc nd_Ferroin]# echo "-j 1" | grep -oE '-j *[[:digit:]]+' | tr -d '-j '
tr: invalid option -- 'j'
Try 'tr --help' for more information.
grep: invalid option -- 'j'
Usage: grep [OPTION]... PATTERNS [FILE]...
Try 'grep --help' for more information.
```

- after

```cmd
[pc nd_Ferroin]# echo "-j 1" | grep -oE '\-j *[[:digit:]]+' | tr -d '\-j '
1
```